inspirehep/modules/records/receivers.py

```diff
@@ -170,20 +175,106 @@ def populate_bookautocomplete(sender, json, *args, **kwargs):
     input_values.extend(titles)
     input_values = [el for el in input_values if el]
 
-    ref = get_value(json, 'self.$ref')
+    record = get_value(json, 'self.$ref', '')
 
     json.update({
         'bookautocomplete': {
             'input': input_values,
             'payload': {
                 'authors': authors,
-                'id': ref,
+                'id': record,
                 'title': titles,
             },
         },
     })
 
 
+def populate_book_series_suggest(sender, json, *args, **kwargs):
```

Comment:
there seems to be a book series autocomplete in the submission forms. It is broken however, as it autocompletes from all journals, instead of only those having `book_series=True` combined with the book series in Literature records. Can you check whether you can salvage that one instead of creating a new one?

Reply:
@michamos I would say that we should probably try to use this one in oreder to have consistent suggesters and remove the old one. @jacquerie, @harunurhan  ?